shell/browser/extensions/electron_messaging_delegate.cc

```diff
@@ -48,6 +47,13 @@ ElectronMessagingDelegate::MaybeGetTabInfo(content::WebContents* web_contents) {
       auto tab = std::make_unique<base::DictionaryValue>();
       tab->SetWithoutPathExpansion(
           "id", std::make_unique<base::Value>(api_contents->ID()));
+      tab->SetWithoutPathExpansion(
+          "url", std::make_unique<base::Value>(api_contents->GetURL()));
+      tab->SetWithoutPathExpansion(
+          "title", std::make_unique<base::Value>(api_contents->GetTitle()));
+      tab->SetWithoutPathExpansion(
+          "audible",
+          std::make_unique<base::Value>(api_contents->IsCurrentlyAudible()));
       return tab;
```

Comment:
Still, it doesn't contain all of the `tabs.Tab` properties, since they are rather related to the UI. I suggest sending a sync IPC message `extensions-get-tab` to allow users to add their own properties to `tabs.Tab` object, for example `active`, `highlighted` etc.